resources/process-openmaptiles.lua

```diff
@@ -458,7 +459,7 @@ end
 function SetEleAttributes(obj)
     local ele = obj:Find("ele")
 	if ele ~= "" then
-		local meter = tonumber(ele) or 0
+		local meter = math.floor(ele) or 0
```

Comment:
I think we probably need to do `math.floor(tonumber(ele) or 0)` here - I've been bitten in the past when there are non-numeric values in the `ele` field:

https://taginfo.openstreetmap.org/tags/ele=%3E160

    > print(math.floor(">160") or 0)
    stdin:1: bad argument #1 to 'floor' (number expected, got string)
    > print(math.floor(tonumber(">160") or 0))
    0

Reply:
Make sense! Also some contributor would tag `ele` like: `ele:160m`.